### PR TITLE
Split policy reform and economic assumption parameters into two separate JSON files

### PR DESCRIPTION
**Pull request by martinholmer**
This pull request implements in Tax-Calculator one of the key ideas in the issue #1140 conversation: the separation of policy reform parameters and economic assumption parameters into two different JSON files.

### CODE REVIEW

**Comment by talumbau** [comment]
This PR would seem to significantly change this interface:

www.ospc.org/taxbrain/file

The effort level is not low for this change on the web application and I am not sure I see the benefit. Is this an "example" PR or is there genuine interest in merging this to master?

**Comment by martinholmer** [comment]
@talumbau said about PR #1148:
> This PR would seem to significantly change this interface:
>
> www.ospc.org/taxbrain/file
>
> The effort level is not low for this change on the web application and I am not sure I see the benefit. Is this an "example" PR or is there genuine interest in merging this to master?

As #1148 mentions, the rationale for making this change in Tax-Calculator, and the additional necessary changes to the [TaxBrain file upload page](www.ospc.org/taxbrain/file), have been discussed at length in issue #1140.  So, yes, "there is genuine interest in merging this to master".

**Comment by MattHJensen** [comment]
@talumbau, there is a lot of material in #1140. Here are the relevant comments for this PR. 

https://github.com/OpenSourcePolicyCenter/taxpuf/pull/1#issuecomment-273926967
https://github.com/open-source-economics/Tax-Calculator/issues/1140#issuecomment-272447197
https://github.com/open-source-economics/Tax-Calculator/issues/1140#issuecomment-272539788

File: taxcalc/tests/test_dropq.py [hunk 1]
```diff
@@ -33,10 +33,24 @@
   "behavior": {
     "_BE_sub": {"2016": [0.25]}
   },
+  "consumption": {
```
Comment (talumbau) [inline taxcalc/tests/test_dropq.py]: Is it correct that REFORM_CONTENTS should have behavior, consumption, and growth? Or should that be only in ASSUMP_CONTENTS?

Comment (martinholmer) [inline taxcalc/tests/test_dropq.py]: Yes, should be in ASSUMP_CONTENTS.  Thanks for finding this problem, which I've fixed in commit 4e5af19.

**Comment by MattHJensen** [comment]
This looks great, @martinholmer.

**Comment by martinholmer** [comment]
Both @MattHJensen and @talumbau have commented on pull request #1148.

Unless anybody expresses concerns, #1148 will be merged into the master branch at the end of the work day on Tuesday, January 24th.

@feenberg @Amy-Xu @andersonfrailey @GoFroggyRun @zrisher @codykallen

**Comment by MattHJensen** [comment]
@martinholmer, do you mind merging this early today instead of late today? I'm hoping to refactor #1135  today.

**Comment by martinholmer** [comment]
@MattHJensen said:
> Do you mind merging this early today instead of late today? I'm hoping to refactor #1135 today.

Merging #1148 now.

**Comment by MattHJensen** [comment]
Thanks!

**Pull request merged**